bin/build-locales

```diff
@@ -204,8 +203,9 @@ async.forEachLimit(views, 5, function (view, cb) {
         fs.readFile(translationsFile, 'utf8', function (err, data) {
             if (err) {
                 if (err.code === 'ENOENT') {
+                    // ignore missing files for english, Meow, and Edible Scratch
                     if (isoCode !== 'en') {
-                        if (defaultLocales.hasOwnProperty(view)) {
+                        if (isoCode !== 'cat' && isoCode !== 'yum'  && defaultLocales.hasOwnProperty(view)) {
```

Comment:
No, we could remove the check/warning. `cat` and `yum` are special cases and we're not going to add the json files, so they will always be missing and we don't need to warn. Everything else should really be there, so we shouldn't see the message except for l10n files that are brand-new and not in Transifex yet. Maybe it would be better to figure out a test that would check for any missing l10n files (ignoring cat and yum).